demos/ws-bootable-jar-example/pom.xml

```diff
@@ -0,0 +1,340 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.jboss.intersmash</groupId>
+        <artifactId>intersmash-demos</artifactId>
+        <version>0.0.1-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+    <artifactId>ws-bootable-jar-example</artifactId>
+    <packaging>pom</packaging>
+
+    <modules>
+        <module>jaxws</module>
+     </modules>
+
+     <properties>
+
+         <version.jakarta.ws.rs-api>3.1.0</version.jakarta.ws.rs-api>
+         <version.resteasy-bom>6.2.4.Final</version.resteasy-bom>
+
+         <version.resteasy-client-api>6.2.4.Final</version.resteasy-client-api>
+
+         <version.jakarta.inject-api>1.0.5</version.jakarta.inject-api>
+         <version.jakarta.jakartaee-api>10.0.0</version.jakarta.jakartaee-api>
+         <version.microprofile-config-api>1.3</version.microprofile-config-api>
+
+         <wildfly-maven-plugin.version>4.0.0.Final</wildfly-maven-plugin.version>
+         <version.maven-war-plugin>3.3.2</version.maven-war-plugin>
+         <!--
+            Default version for the Bootable JAR Plugin is set here and can be overridden,
+            e.g. also for pulling the productized version
+        -->
+         <version.wildfly-jar-maven-plugin>8.0.0.Final</version.wildfly-jar-maven-plugin>
+
+         <version.wildfly.feature-pack>27.0.0.Final</version.wildfly.feature-pack>
+         <version.wildfly-cloud-galleon-pack>2.0.0.Final</version.wildfly-cloud-galleon-pack>
+
+         <version.jakarta.xml.ws-api>4.0.0</version.jakarta.xml.ws-api>
```

Comment:
Could you please just move this up [here](https://github.com/Intersmash/intersmash/pull/83/files#diff-5b3ab062199625db1c77ee143c5340b7afd3cad261ac5018db159ea3cd05781dR25)?